doc/rbd/rbd-openstack.rst

```diff
@@ -215,11 +215,22 @@ Edit ``/etc/glance/glance-api.conf`` and add under the ``[glance_store]`` sectio
     rbd_store_ceph_conf = /etc/ceph/ceph.conf
     rbd_store_chunk_size = 8
 
+.. important:: Glance has not completely moved to 'store' yet.
+    So we still need to configure the store in the DEFAULT section until Kilo.
 
-For more information about the configuration options available in Glance please see: http://docs.openstack.org/trunk/config-reference/content/section_glance-api.conf.html.
+Kilo
+~~~~
+Edit ``/etc/glance/glance-api.conf`` and add under the ``[glance_store]`` section::
 
-.. important:: Glance has not completely moved to 'store' yet.
-    So we still need to configure the store in the DEFAULT section.
+    [glance_store]
+    stores = rbd
+    default_store = rbd
+    rbd_store_pool = images
+    rbd_store_user = glance
+    rbd_store_ceph_conf = /etc/ceph/ceph.conf
+    rbd_store_chunk_size = 8
+
+For more information about the configuration options available in Glance please see: http://docs.openstack.org/trunk/config-reference/content/section_glance-api.conf.html.
 
```

Comment:
May be update this URL too? It looks like it is outdated (redirecting to http://docs.openstack.org/mitaka/config-reference/index.html).

For Kilo I see documentation:

http://docs.openstack.org/kilo/config-reference/content/ch_configuring-openstack-image-service.html

specifically:

http://docs.openstack.org/kilo/config-reference/content/configuring-image-service-backends.html

The latest (Mitaka):

http://docs.openstack.org/mitaka/config-reference/image-service.html
http://docs.openstack.org/mitaka/config-reference/image-service/backends.html